docs/Maintainer-Guidelines.md

```diff
@@ -9,6 +9,10 @@ Maybe you were looking for the [Formula Cookbook](Formula-Cookbook.md)?
 
 This document is current practice. If you wish to change or discuss any of the below: open a PR to suggest a change.
 
+## Mission
+
+Homebrew seeks to be the missing package manager for macOS. It's primary goal is to be useful. It should be as useful as possible to as many people as possible that we can maintain as a small group of volunteers to a commercially high standard. Where possible and sensible it should seek to use macOS features and blend in with the macOS and Apple ecosystem.
```

Comment:
I've incorporated others' suggestions and added a few edits of my own.
```suggestion
Homebrew aims to be the missing package manager for macOS, whose primary goal is to be useful. It aims to be as useful as possible to as many people as possible, while being maintained by a small group of volunteers to a high standard of quality. Where possible and sensible it should seek to use features of macOS to blend in with the macOS ecosystem.
```

Reply:
Thanks all, have gone with a blend of @zbeekman and @sjackman's versions.

> Also, might it me wise to remove the “commercially”? On commercial apps support is supposed to always be nice, even if the customer is rude.

@vitorgalvao Pedantic perhaps but I'd say that's not a statement about the quality of Homebrew but the quality of Homebrew's support (which, we agree, should not be required to meet a commercial standard).

Reply:
> Pedantic perhaps

Not at all, I agree. I was commenting in the sense of not wanting a rude user throwing that in our faces later on.